regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -32,9 +31,6 @@ var ChildView = Backbone.View.extend({
                 }
 
                 this.route(this.options);
-
-                GAEvents.sendEvent('section:open', this.options.id);
-
```

Comment:
This change makes a lot of sense — better to track the explicit user action than the side-effect — but it's interesting to me that this gets called twice.